pennylane/qchem/givens_decomposition.py

```diff
@@ -38,26 +36,27 @@ def _givens_matrix(a, b, left=True, tol=1e-8):
         tol (float): determines tolerance limits for :math:`|a|` and :math:`|b|` under which they are considered as zero.
 
     Returns:
-        np.ndarray (or tensor): Givens rotation matrix
+        qml.math.array (or tensor): Givens rotation matrix
```

Comment:
Not a big problem as this docstring isn't user facing. But do we still need `(or tensor)` in this case?

Also, [`qml.math.array`](https://docs.pennylane.ai/en/stable/code/api/pennylane.math.array.html) is not itself a type, it returns a tensor-like quantity depending on its interface. We've generally used `tensor_like` for this - see, e.g., [`reduce_dm`](https://docs.pennylane.ai/en/stable/code/api/pennylane.math.reduce_dm.html).

Reply:
Changed